src/main/java/teammates/sqllogic/core/CoursesLogic.java

```diff
@@ -121,6 +122,17 @@ public void deleteCourseCascade(String courseId) {
             return;
         }
 
+        usersLogic.deleteStudentsInCourseCascade(courseId);
+        List<FeedbackSession> feedbackSessions = fsLogic.getFeedbackSessionsForCourse(courseId);
+        feedbackSessions.forEach(feedbackSession -> {
+            fsLogic.deleteFeedbackSessionCascade(feedbackSession.getName(), courseId);
+        });
+        coursesDb.deleteSectionsByCourseId(courseId);
```

Comment:
Is this not already handled by the `cascade` in the `OneToMany` annotation in `Course` yet?